setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
```

It's better to explicitly request the generated wheel to be universal:
```suggestion
[bdist_wheel]
universal = 1

[metadata]
```